pennylane/gradients/finite_difference.py

```diff
@@ -317,9 +313,9 @@ def finite_diff(
         >>> gradient_tapes, fn = qml.gradients.finite_diff(tape)
         >>> gradient_tapes
         [<QuantumTape: wires=[0], params=3>,
-         <QuantumTape: wires=[0], params=3>,
-         <QuantumTape: wires=[0], params=3>,
-         <QuantumTape: wires=[0], params=3>]
+         <QuantumScript: wires=[0], params=3>,
+         <QuantumScript: wires=[0], params=3>,
+         <QuantumScript: wires=[0], params=3>]

```

Comment:
I don't think it's technically inconsistent since `QuantumTape` is a child of `QuantumScript`, but yes does it look bad/confusing? :thinking: Overall, I think it's fine for now, let's see if someone is confused!